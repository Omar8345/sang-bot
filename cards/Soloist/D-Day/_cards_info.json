{
  "EDDAGD": {
    "name": "AUGUST D",
    "rarity": "E"
  },
  "BDDAGD": {
    "name": "AUGUST D",
    "rarity": "B"
  },
  "CDDAGD": {
    "name": "AUGUST D",
    "rarity": "C"
  },
  "DDDAGD": {
    "name": "AUGUST D",
    "rarity": "D"
  },
  "ADDAGD": {
    "name": "AUGUST D",
    "rarity": "A"
  }
}