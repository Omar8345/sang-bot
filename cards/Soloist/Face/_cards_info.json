{
  "EFCJMN": {
    "name": "JIMIN",
    "rarity": "E"
  },
  "BFCJMN": {
    "name": "JIMIN",
    "rarity": "B"
  },
  "CFCJMN": {
    "name": "JIMIN",
    "rarity": "C"
  },
  "DFCJMN": {
    "name": "JIMIN",
    "rarity": "D"
  },
  "AFCJMN": {
    "name": "JIMIN",
    "rarity": "A"
  }
}