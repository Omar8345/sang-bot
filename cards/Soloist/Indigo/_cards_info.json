{
  "EIGRMM": {
    "name": "RM",
    "rarity": "E"
  },
  "BIGRMM": {
    "name": "RM",
    "rarity": "B"
  },
  "CIGRMM": {
    "name": "RM",
    "rarity": "C"
  },
  "DIGRMM": {
    "name": "RM",
    "rarity": "D"
  },
  "AIGRMM": {
    "name": "RM",
    "rarity": "A"
  }
}