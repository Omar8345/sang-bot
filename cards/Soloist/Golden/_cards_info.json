{
  "EGDJGK": {
    "name": "JUNGKOOK",
    "rarity": "E"
  },
  "BGDJGK": {
    "name": "JUNGKOOK",
    "rarity": "B"
  },
  "CGDJGK": {
    "name": "JUNGKOOK",
    "rarity": "C"
  },
  "DGDJGK": {
    "name": "JUNGKOOK",
    "rarity": "D"
  },
  "AGDJGK": {
    "name": "JUNGKOOK",
    "rarity": "A"
  }
}